app/components/tables/headers/sort.js

```diff
@@ -35,6 +41,13 @@ export default class Sort extends Component {
           sortDir : null
         });
       }
+      else {
+        //default sorting for starts-at
```

Comment:
Codacy found an issue: [Expected space or tab after '//' in comment.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6279483)